src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -117,6 +145,28 @@ public function register_routes() {
 		);
 	}
 
+	/**
+	 * Returns the fallback template for a given slug.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param WP_REST_Request $request The request instance.
+	 *
+	 * @return WP_REST_Response|WP_Error
+	 */
+	public function get_template_fallback( $request ) {
+		if ( empty( $request['slug'] ) ) {
+			return new WP_Error(
+				'rest_invalid_param',
+				__( 'Invalid slug.', 'gutenberg' ),
```

Comment:
Remove `'gutenberg'` text domain.
```suggestion
				__( 'Invalid slug.' ),
```